### PR TITLE
Adding port, user, password and host params

### PR DESCRIPTION
**Pull request by ankit-jha**

### CODE REVIEW

File: scripts/load_timescaledb.sh [hunk 1]
```diff
@@ -10,7 +10,10 @@ fi
 # Load parameters - common
 DATA_FILE_NAME=${DATA_FILE_NAME:-timescaledb-data.gz}
 DATABASE_USER=${DATABASE_USER:-postgres}
-
+DATABASE_NAME=${DATABASE_NAME:-astimescaledb}
+DATABASE_HOST=${DATABASE_HOST:-192.168.64.3}
+DATABASE_PORT=${DATABASE_PORT:-30823}
+DATABASE_PWD=${DATABASE_PWD:-aef31a17aae11d04181d5d009}
```
Comment (jonatas) [inline scripts/load_timescaledb.sh:16]: Please, do not use your personal values as default here. Instead, let's use the defaults from postgres.

```suggestion
DATABASE_NAME=${DATABASE_NAME:-benchmark}
DATABASE_HOST=${DATABASE_HOST:-localhost}
DATABASE_PORT=${DATABASE_PORT:-5432}
DATABASE_PWD=${DATABASE_PWD:-password}
```

**Review by jonatas — CHANGES_REQUESTED** [review]
@ankit-jha thanks for the contribution and sorry for being late here. Would you mind bringing generic values instead of your custom config to the defaults?

We also need to resolve the conflicts in the branch, rebasing it at the top of the master branch. Can you provide the updates?

**Review by ankit-jha — COMMENTED** [review]
@jonatas 
Thank you for the review

**Review by ankit-jha — COMMENTED** [review]
Requested changes done

File: scripts/load_timescaledb.sh (hunk 1 above)
Comment (ankit-jha) [reply scripts/load_timescaledb.sh]: Updated

**Review by jonatas — APPROVED** [review]
Thanks, @ankit-jha! Probably @RobAtticus or @antekresic can merge it in the next few days!

**Pull request merged**